scripts/qa_crowbarsetup.sh

```diff
@@ -2337,7 +2337,7 @@ function oncontroller_testsetup()
         crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
         pushd /var/lib/openstack-tempest-test
         echo 1 > /proc/sys/kernel/sysrq
-        ./run_tempest.sh -N $tempestoptions 2>&1 | tee tempest.log
+        testr run $tempestoptions | tee tempest.log | subunit-2to1 | tools/colorizer.py
```

Comment:
Yes, this is the first commit of run_tempest.sh:
https://github.com/openstack/tempest/commit/61f7d5ece5fa5f237ff692f585f5f8e0a6448bb5

In L98 we can see the is basically the same call